src/app/info/feedback/feedback-form/feedback-form.component.html

```diff
@@ -3,7 +3,7 @@
     <form class="primary" [formGroup]="feedbackForm" (ngSubmit)="createFeedback()">
       <h1>{{ 'info.feedback.head' | translate }}</h1>
       <p>{{ 'info.feedback.info' | translate }}</p>
-      <fieldset class="col p-0">
+      <form class="col p-0">
```

Comment:
I cannot get the feedback form the work anymore?  It appears this `<form>` is nested under another `<form>` on line 3.  Maybe that's messing things up?

If I visit the feedback form, it loads.  But the "Send Feedback" button will not enable, even if you enter in an email and comments.